app/frontend/src/store/modules/form.js

```diff
@@ -153,11 +163,33 @@ export default {
     },
   },
   actions: {
+    async getForm({ commit }, ipcPlanId) {
+      commit('setGettingForm', true);
+      commit('setGetFormError', '');
+      try {
+        const response = await ipcService.getIPCContent(ipcPlanId);
+        if (!response.data) {
+          throw new Error(`Failed to GET for ${ipcPlanId}`);
+        }
+        const data = response.data;
+
+        commit('updateIpcPlan', data.ipcPlan);
+        commit('updateBusiness', data.business);
+        commit('updateContacts', data.contacts[0]);
+        commit('updateCovidContact', data.covidContact);
+        commit('updateLocation', data.location);
+        commit('setSubmissionComplete');
+      } catch (error) {
+        console.error(`Error getting form: ${error}`); // eslint-disable-line no-console
+        commit('setGetFormError', 'An error occurred while attempting to get the IPC details. Please refresh and again.');
```

Comment:
You forgot the word try after the and.
Maybe we can just say "An error occurred while attempting to fetch details. Please refresh and try again". Don't really have to say IPC here do we (as its implicit)